Formula/canu.rb

```diff
@@ -0,0 +1,24 @@
+class Canu < Formula
+  desc "Single molecule sequence assembler"
+  homepage "https://canu.readthedocs.org/en/latest/"
+  url "https://github.com/marbl/canu/archive/v1.6.tar.gz"
+  sha256 "470e0ac761d69d1fecab85da810a6474b1e2387d7124290a0e4124d660766498"
+  head "https://github.com/marbl/canu.git"
+  # cite "10.1038/nbt.3238"
+
+  # Fix fatal error: 'omp.h' file not found
+  depends_on "gcc"
+
+  def install
+    system "make", "-C", "src"
+    arch = Pathname.new(Dir["*/bin"][0]).dirname
+    rm_r arch/"obj"
+    prefix.install arch
+    bin.install_symlink prefix/arch/"bin/canu"
+    doc.install Dir["README.*"]
+  end
+
+  test do
+    system "#{bin}/canu", "--version"
```

Comment:
Please modify this test to use `assert_match` and `shell_output`. You can see examples in other formulae.